packages/remix-dev/compiler/assets.ts

```diff
@@ -87,7 +87,7 @@ export async function createAssetsManifest(
       );
       let route = routesByFile.get(entryPointFile);
       invariant(route, `Cannot get route for entry point ${output.entryPoint}`);
-      let sourceExports = await getRouteModuleExportsCached(config, route.id);
+      let sourceExports = await getRouteModuleExports(config, route.id);
```

Comment:
why aren't we using the cached one?

Reply:
We are. I renamed `getRouteModuleExportsCached` to just `getRouteModuleExports` since we only ever use the cached version.